tests/publiccloud/upload_image.pm

```diff
@@ -33,7 +33,7 @@ sub run {
     }
 
     # Download the given image via wget. Note that by default wget retries 20 times before giving up
-    my $cmd = "wget --no-check-certificate --retry-connrefused --retry-on-host-error $img_url -O $img_name";
+    my $cmd = "wget -q --no-check-certificate --retry-connrefused --retry-on-host-error $img_url -O $img_name";
```

Comment:
This runs on our PC helper instance which is SLES.

Reply:
No but we control the image ourselves and it is only one for every test run. Because we can control the tools we need here we don't need to limit ourselves.

Reply:
yy, i understand

Reply:
The tools image is created [here](https://openqa.suse.de/tests/overview?distri=sle&version=15-SP3&build=0045&groupid=276) and is used in all `upload_img` jobs. All of those jobs use the same HDD, regardless of the SLES version because it's just a helper VM.